defaults/main.yml

```diff
@@ -1,14 +1,23 @@
 ---
 varnish_package_name: "varnish"
 varnish_version: "4.1"
+# Varnish 5+ is currently only supported on Debian and Ubuntu.
+# varnish_version: "5.1"
 
 varnish_use_default_vcl: true
 varnish_default_vcl_template_path: default.vcl.j2
 
 varnish_default_backend_host: "127.0.0.1"
 varnish_default_backend_port: "8080"
+# If backend server expect PROXY protocol connexion:
+# varnish_default_backend_proxy_proto: 1
+
+varnish_listen_ports:
+  - port: 80
+  - port: 6081
+    # If Varnish must expect a PROXY protocol connexion:
+    proxy_proto: yes
```

Comment:
It seems there's a lot of checking going on in all the places where `proxy_proto` is being used. To make this easier to review and include... I hate to ask, but could you separate this PR into two—the first PR to make only the necessary changes to get Varnish 5.x installed, and the second PR to add changes to get `proxy_proto` supported as an option?

The main reason I ask is because I would want to do more testing around it if adding it—and also since it changes some of the basic variables it would mean a major version bump for this role, and I usually like to package up multiple big changes if possible so I don't have to do a major version bump on every commit :)